parachains/README.md

```diff
@@ -0,0 +1,22 @@
+# Parachains
+
+This directory is the home of Parity-developed parachain runtimes. This directory is _runtime
+focused,_ and does not include builds of parachain _nodes._
```

Comment:
```suggestion
focused_, and does not include builds of parachain _nodes._
```

Reply:
Is it a normal pattern to put punctuation outside of formatting markup? I really don't know, just always put punctuation in the same rendering as whatever it's adjacent to since some sites shift things left/right and characters can appear to overlap when adjacent characters have different formatting.

Reply:
I'd have the punctuation outside of the markup where possible. _, seems better to me.

Reply:
I would not adjust to some sites having suboptimal renderings for punctuation and moving punctuation into formatting scopes. But I don't have a strong opinion, just stroke me as a bit of an oddity.